Tests/StreamChatTests/Query/Sorting/ListDatabaseObserver+Sorting.swift

```diff
@@ -0,0 +1,301 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+@testable import StreamChat
+@testable import StreamChatTestTools
+import XCTest
+
+final class ChannelList_Sorting_Tests: XCTestCase {
+    let dateNow = Date()
+    let datePast = Date().addingTimeInterval(-60 * 60 * 3)
+    let dateWayPast = Date().addingTimeInterval(-60 * 60 * 100)
+
+    lazy var defaultChannels = [
+        (name: "A", createdAt: datePast),
+        (name: "B", createdAt: dateNow),
+        (name: "C", createdAt: dateWayPast),
+        (name: "D", createdAt: dateNow),
+        (name: "E", createdAt: datePast)
+    ]
+
+    lazy var bulkChannels: [(name: String, createdAt: Date)] = (0...50).map { _ in
```

Comment:
nit: Can we increases this one to 1000? Having a sample as small as 50 it will only highlight a really bad issue (e.g cubic or quadratic complexity). With bigger sample we should be able to identify even smaller issues.

Reply:
You are right, but only increasing from 50 to 100 adds 15 seconds, with a total of 25s. We cannot afford to spend that much time on PR checks.
In this particular case, given the logic, I would say it is safe to leave it as is. Thoughts?